examples/mlflow_example.py

```diff
@@ -67,10 +67,10 @@ def eval_input_fn():
         ))
     return Experiment(
         estimator,
-        tf.estimator.TrainSpec(train_input_fn, max_steps=10000),
+        tf.estimator.TrainSpec(train_input_fn, max_steps=100),
```

Comment:
Why change of max steps

Reply:
I know it is not related to this review but I don't want to do another want. It is to align all tests with 100 steps. It is enough to get the code running but reasonable fast for just running the samples as integration tests.